.github/workflows/verifyAudit.yml

```
@@ -154,17 +162,23 @@ jobs:
-              console.log(`Label 'AuditCompleted' is currently not assigned`);
+              console.log(`${colors.green}Label 'AuditCompleted' is currently not assigned${colors.reset}`);
+
+            // set CONTINUE to true to make sure the following steps are executed
+            core.exportVariable('CONTINUE', 'true');
 
       - name: Check Audit Log
         continue-on-error: true
         id: check-audit-log
-        if: env.AUDIT_REQUIRED == 'true'
+        if: ${{ always() && env.AUDIT_REQUIRED == 'true' && env.CONTINUE == 'true' }} # always() ensures that validation is always executed, even if env variable is not set
         run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2034:warning:12:1: COMMIT_HASHES_FILE appears unused. Verify use (or export if used externally)

(shellcheck)